sled-agent/src/backing_fs.rs

```diff
@@ -49,16 +53,19 @@ struct BackingFs {
     compression: Option<&'static str>,
     // Linked service
     service: Option<&'static str>,
+    // Subdirectories to ensure
+    subdirs: Option<&'a [&'static str]>,
```

Comment:
This is completely fine as-is, but I'll just note that you could do `Option<&'static [&'static str]>` here to avoid the `<'a>` bound on `BackingFs`. (Doing so would be less flexible than what you have, but since everything else in here has a `'static` bound we could make this match.)